### PR TITLE
Bring in TACo commits

### PR DESCRIPTION
**Pull request by theref**
This PR brings in all the missing TACo commits and resolves the merge conflicts

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,14 +9,15 @@ A web forum demo application built on OrbisDB.
...
+**TACo** - A decentralized access control layer, through which decryption rights to non-public forum content (posts, comments, reactions, etc.) are verified and enforced by a decentralized cohort of TACo nodes. 
 **ThirdWeb SDK** - Uses their IPFS upload feature for image file storage
 
 ## Installation
 
 First, create a local clone of the application:
 
 ```bash
-git clone https://github.com/ceramicstudio/deforum && cd deforum
+git clone https://github.com/nucypher/taco-orbisdb-demo && cd taco-orbisdb-demo
```
Comment (derekpierre) [inline README.md]:
```suggestion
git clone https://github.com/nucypher/deforum-taco && cd deforum-taco
```

**Review by manumonti — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (manumonti) [reply README.md]: Actually, if this is going to be merged eventually on `ceramicstudio/deforum-taco`, should not be this the URL?

`git clone https://github.com/ceramicstudio/deforum-taco`

Comment (derekpierre) [reply README.md]: Good point - although the README in `nucypher/deforum-taco` probably shouldn't say that.

Perhaps it is better to make this more flexible and just say something akin to "clone this repos".

Comment (theref) [reply README.md]: done

**Pull request merged**